azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,349 @@
-2. Select **Edit and deploy** along the top.
+2. Click **Edit and deploy**.
 3. Put a check in the box next to your **op-scim-bridge** container and select **Edit**.
 4. Change the version number **2.9.0** in the **Image and Tag** field, **1password/scim:v2.9.0** to match the latest version from our [SCIM Bridge Release Notes page](https://app-updates.agilebits.com/product_history/SCIM).
 5. Select **Save**.
 6. Select **Create** to deploy a new revision using the updating image.
 7. Log into your SCIM bridge URL with your bearer token to validate in the top left hand side that you are running the version of the SCIM Bridge. (logging in with the bearer token will also update your Automated User Provisioning page with the latest access time and with the current version).
 
-## Resource recommendations
+## Appendix: Resource recommendations
 
-The 1Password SCIM Bridge Pod should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
+The pod for 1Password SCIM Bridge should be vertically scaled you provision a large number of users or groups. These are our default resource specifications and recommended configurations for provisioning at scale:
 
 | Volume    | Number of users | CPU   | memory |
 | --------- | --------------- | ----- | ------ |
 | Default   | <1,000          | 0.25  | 0.5Gi  |
 | High      | 1,000–5,000     | 0.5   | 1.0Gi  |
 | Very high | >5,000          | 1.0   | 1.0Gi  |
 
-If provisioning more than 1,000 users, the resources assigned to the SCIM bridge container should be updated as recommended in the above table. The resources specified for the Redis container do not need to be adjusted.
+If you're provisioning more than 1,000 users, update the resources assigned to the SCIM bridge container to follow these recommendations. The resources specified for the Redis container do not need to be adjusted.
 
-#### Default
+### Default
 
-To revert to the default specification that is suitable for provisioning up to 1,000 users:
+To revert to the default specification that is suitable for provisioning up to 1,000 users, run the following command:
 
 ```sh
 az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
   --cpu 0.25 --memory 0.5Gi
 ```
 
-#### High volume deployment
+### High volume deployment
 
-For provisioning up to 5,000 users:
+If you're provisioning up to 5,000 users:
 
 ```sh
 az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
   --cpu 0.5 --memory 1.0Gi
 ```
 
-#### Very high volume
+### Very high volume
 
-For provisioning more than 5,000 users:
+If you're provisioning more than 5,000 users:
 
 ```sh
 az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
   --cpu 1.0 --memory 1.0Gi
 ```
 
```

Comment:
I am not 100% sold on removing the Azure Cloud Shell method for connecting google workspace completely. I feel we need to keep this info somewhere. Perhaps as additional appendix related info. We do call out how to upgrade resources in the appendix, but only show the cloud shell method to do so (which I am fine with).

Reply:
@Penderworth I feel that from our team, we are trying to ensure we are documenting in GH the ability to scale the SCIM bridge to meet higher demand requirements (if required be the customer). While this may not be the best place in the GUI readme, I think some items such as resources requirements and get help (troubleshooting tips) might be great in another md file with this info. That way it is available for both the GUI deployment method and the Cloud shell method in AKB.

Reply:
That's a good point. I'll add it back and update it to match the rest of the style here!

Reply:
@black-bryan Alright these are back in https://github.com/1Password/scim-examples/pull/290/commits/ac46bcbd76545ce42c5c8242722d5b47b6bff64b. Let me know how they look.